dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Shunts/SectionControl.mo

```diff
@@ -0,0 +1,53 @@
+within Dynawo.Electrical.Controls.Shunts;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model SectionControl "Section control for shunts with sections. The section of the shunt is changed depending on a measured voltage that is compared to its reference."
+  import Dynawo.Connectors;
+  import Dynawo.Types;
+  import Modelica;
+
+  Connectors.ZPin URefPu(value(start = URef0Pu)) "Voltage regulation set point in p.u (base UNom)";
```

Comment:
Yes, I put a ZPin to be able to change it if we want to. But maybe it is not necessary. @marcochiaramello what do you think ?

Reply:
We can wait for @marcochiaramello opinion on this, but if you want to keep it as a connector it would be better to use a ImPin which is more generic and enables all kind of changes (whereas a ZPin would be for discrete changes)

Reply:
The thing I like with the ZPin is that if I don't connect anything (no step or setpoint), it works as if it is a parameter and takes URef0. I think that with a ImPin I will have an error if I don't have a step or a setpoint in the .dyd, no ?

Reply:
Yes, it would require something in the preassembled model...

Reply:
It works with ImPin and discrete in the extvar, thanks !

Reply:
No Pb! ;)